jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/websocket/WebSocketConstants.java

```diff
@@ -29,10 +29,10 @@
 
     public static final String DIAGNOSTIC_ERR_MSG = "Cannot calculate WebSocket diagnostics";
 
-    public static final String DIAGNOSTIC_PATH_PARAMS_ANNOT_MISSING = "Parameter is missing annotation @PathParams.";
+    public static final String DIAGNOSTIC_PATH_PARAMS_ANNOT_MISSING = "Parameters of type String, any Java primitive types or boxed version thereof must be annotated with @PathParams.";
```

Comment:
I think in this case it should mention `annotated with @PathParams` as this diagnostic message is related to these specific types missing the necessary `@PathParam` annotation. Those parameter types are not considered valid unless they have that annotation. Otherwise, the user will not know why that error is showing

Reply:
Yes I agree, sorry full message should be:

    public static final String DIAGNOSTIC_PATH_PARAMS_ANNOT_MISSING = "Parameters of type String, any Java primitive type, or boxed version thereof must be annotated with @PathParams.";

Reply:
will do that, thanks!